### PR TITLE
Update to latest modelserver version and fix welcome page menu entry

### LINKED ISSUE #329: Fix textual DSL (wfconfig editor)
After upgrading to the latest modelserver version (PR #328) , the textual DSL alias the wfconfig file support stopped working.
~~As the modelserver got more strict on loading resources on startup, we have to make sure that all necessary resources can be loaded correctly.  (e.g. check if WorkflowConfiguration needs to be registered as EPackageConfiguration)~~

Update: Please see https://github.com/eclipse-emfcloud/emfcloud-modelserver/issues/71 for details

```
Caused by: java.lang.NoClassDefFoundError: org/eclipse/core/resources/ResourcesPlugin
	at org.eclipse.emf.ecore.plugin.EcorePlugin.getWorkspaceRoot(EcorePlugin.java:1149)
	at org.eclipse.emf.ecore.resource.impl.PlatformResourceURIHandlerImpl.getWorkspaceRoot(PlatformResourceURIHandlerImpl.java:488)
	at org.eclipse.emf.ecore.resource.impl.PlatformResourceURIHandlerImpl.<clinit>(PlatformResourceURIHandlerImpl.java:456)
	at org.eclipse.emf.ecore.resource.URIHandler.<clinit>(URIHandler.java:51)
	at org.eclipse.emf.ecore.resource.impl.ExtensibleURIConverterImpl.<init>(ExtensibleURIConverterImpl.java:138)
	at org.eclipse.emf.ecore.resource.impl.ResourceSetImpl.getURIConverter(ResourceSetImpl.java:499)
	at org.eclipse.emf.ecore.resource.impl.ResourceSetImpl.getResource(ResourceSetImpl.java:369)
	at org.eclipse.emfcloud.modelserver.common.codecs.XmiCodec.decode(XmiCodec.java:76)
	at org.eclipse.emfcloud.modelserver.common.codecs.XmiCodec.decode(XmiCodec.java:57)
	at org.eclipse.emfcloud.modelserver.common.codecs.XmiCodec.decode(XmiCodec.java:51)
	at org.eclipse.emfcloud.modelserver.client.ModelServerClient.decode(ModelServerClient.java:678)
	at org.eclipse.emfcloud.modelserver.client.ModelServerClient.lambda$2(ModelServerClient.java:114)
	at java.base/java.util.Optional.flatMap(Optional.java:294)
	at org.eclipse.emfcloud.modelserver.client.ModelServerClient.lambda$1(ModelServerClient.java:114)
	at java.base/java.util.function.Function.lambda$andThen$1(Function.java:88)
	at java.base/java.util.function.Function.lambda$andThen$1(Function.java:88)
	at org.eclipse.emfcloud.modelserver.client.Response.body(Response.java:39)
	at com.eclipsesource.coffee.common.ModelServerClientUtil.loadResource(ModelServerClientUtil.java:30)
	at com.eclipsesource.coffee.common.ModelServerClientUtil.loadResource(ModelServerClientUtil.java:34)
	at com.eclipsesource.workflow.dsl.ide.server.WorkflowIndexService.getContent(WorkflowIndexService.java:139)
	at com.eclipsesource.workflow.dsl.ide.server.WorkflowIndexService.workflowFileCreated(WorkflowIndexService.java:119)
```

### PR DESCRIPTION
**Pull request by ndoschek**
- Update to latest modelserver version
- Fix multiple 'Getting started' menu entry bug
  - Use unique widgetId and order for menu entry
- Improve logging for glsp server launcher
- Fix backend launch config to load plugins correctly
- Fix browser-app start scripts
- Add missing eslint script executions in several web folders and fix eslint errors
- Remove deprecated 'eslint.enable setting'
- Add hint for debugging GLSP server

Signed-off-by: Nina Doschek <ndoschek@eclipsesource.com>

Fixes #329

### CODE REVIEW

File: web/browser-app/package.json [hunk 1]
```diff
@@ -45,8 +45,8 @@
   },
   "scripts": {
     "prepare": "yarn run download:plugins && theia build --mode development",
-    "start": "theia start --WORKFLOW_NOTATION_LSP=5008 --port=3000 --loglevel=debug --root-dir=../../backend/examples/ --loglevel=debug  --plugins=local-dir:./plugins",
-    "debug": "theia start --WORKFLOW_NOTATION_LSP=5008 --WF_LSP=5017 --port=3000 --loglevel=debug --root-dir=../../backend/examples/ --loglevel=debug  --plugins=local-dir:./plugins",
+    "start": "theia start --WORKFLOW_NOTATION_LSP=5008 --WF_LSP=5017 --WF_ANALYZER=5083 --port=3000 --root-dir=../../backend/examples/SuperBrewer3000 --plugins=local-dir:./plugins",
```
Comment (eneufeld) [inline web/browser-app/package.json:48]: I would not set all the ports in the start script, as they ignore local jars. This should only be set for debug

**Review by eneufeld — COMMENTED** [review]
Change looks good, I would make sure, that start works without eclipse, only based on the built jars

**Comment by ndoschek** [comment]
Thanks for you review Eugen, I updated my PR and running only with the built jars works on my setup again.
The model server configure requests causes an error in the Theia backend log atm, but I provided a fix for this.

I will ping you again if this fix is merged and I upgraded my PR to the newest modelserver version.

File: web/coffee-workflow-analyzer-editors/src/node/language-contribution.ts [hunk 2]
```diff
@@ -51,12 +54,13 @@ export class WorkflowContribution extends BaseLanguageServerContribution {
             const command = 'java';
 
             const jarPath = this.getJarPath();
-            const args: string[] = ['-jar', jarPath];
+            const args: string[] = ['-jar', jarPath, '-startSocket'];
```
Comment (eneufeld) [inline web/coffee-workflow-analyzer-editors/src/node/language-contribution.ts:57]: startSocket and createProcessStreamConnectionAsync doesn't fit, it is weird it worked for you

**Review by eneufeld — APPROVED** [review]

**Pull request merged**